### PR TITLE
finished refactoring the endpoints

### PR DESCRIPTION
**Pull request by maikefischer**
Helluuu, 

These are the new paths for the following Endpoints:

##Endpoint: Talentmanager
/talentmanager - Posts new talentmanager
/talentmanager/id - Gets or deletes talentmanger
/talentmanager/all - Gets all talentmanagers

##Endpoint: TalentTeam
/talentteam - Posts new talentteam
/talentteam/id - Gets or deletes talentteam
/talentteam/all - Gets all talentteam

##Endpoint: TalentEndpoint
/talent - Posts new talent
/talent/id - Gets or deletes talent
/talent/all - Gets all talent
/talent/id /expenditures - Gets all expenditures

##Endpoint: ExpenditureEndpoint
/expenditure - Gets all expenditures
/expenditure/changestate/{id}/state/{state} - changes state of a specific expenditure

### CODE REVIEW

File: src/main/java/com/nionios/trial/api/ExpenditureEndpoint.java [hunk 1]
```diff
@@ -34,6 +32,18 @@ public Response expendituresInProgress() {
         return Response.ok(expenditures).build();
     }
 
+
+    // change state
+    @PATCH
+    @Path("/changestate/{id}/state/{state}")
```
Comment (cornelisdemooij) [inline src/main/java/com/nionios/trial/api/ExpenditureEndpoint.java]: Can you change this to "/{id}/state/{state}"?

File: src/main/java/com/nionios/trial/api/TalentEndpoint.java [hunk 2]
```diff
@@ -26,22 +19,51 @@
...
+    @POST
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response addTalent(Talent talent){
+        Talent result = talentManagerService.addTalent(talent);
+        return Response.accepted(result).build();
+    }
+
+    @DELETE
+    @Path(value = "/{id}")
```
Comment (cornelisdemooij) [inline src/main/java/com/nionios/trial/api/TalentEndpoint.java]: `value = ` is not necessary in @Path apparently, can you remove that from each @Path?

**Review by cornelisdemooij — CHANGES_REQUESTED** [review]
Some minor changes, see comments on the code.

Can you also clean up the formatting a little bit? Remove superfluous blank lines and make sure the indentation is correct.

File: src/main/java/com/nionios/trial/api/ExpenditureEndpoint.java (hunk 1 above)
Comment (maikefischer) [reply src/main/java/com/nionios/trial/api/ExpenditureEndpoint.java]: Done

File: src/main/java/com/nionios/trial/api/TalentEndpoint.java (hunk 2 above)
Comment (maikefischer) [reply src/main/java/com/nionios/trial/api/TalentEndpoint.java]: Done

**Review by maikefischer — COMMENTED** [review]
Cleaned the code

**Review by cornelisdemooij — APPROVED** [review]
Goed gedaan

**Pull request merged**